src/app/[locale]/selected-filters.tsx

```diff
@@ -0,0 +1,114 @@
+import {SearchResultFilter} from '@/lib/dataset-fetcher';
+import {Dispatch} from 'react';
+import {useTranslations} from 'next-intl';
+import {XMarkIcon} from '@heroicons/react/24/solid';
+
+interface TagProps {
+  name?: string;
+  remove: () => void;
+}
+
+function Tag({name, remove}: TagProps) {
```

Comment:
A question about the name: `Tag`, to me, is what it looks like, not what it does. Would it be possible to use a name that makes its function explicit? If we decide to not use 'tags' anymore, but some other form, we don't have to redefine the component and only reimplement its inner working?

Reply:
I have removed this component. And used the new badge component.